src/main.ts

```diff
@@ -73,7 +73,7 @@ async function bootstrap() {
   });
 
   admin.initializeApp({
-    credential: admin.credential.cert('/firebase_service_account.json'),
+    credential: admin.credential.cert('firebase_service_account.json'),
```

Comment:
Please use config/env here to enable passing different files for different envs (local, dev and live)